src/test/scala/org/broadinstitute/dsde/workbench/sam/service/TosServiceSpec.scala

```diff
@@ -29,82 +33,137 @@ class TosServiceSpec extends AnyFlatSpec with TestSupport with BeforeAndAfterAll
   protected def clearDatabase(): Unit =
     TestSupport.truncateAll
 
-  "TosService" should "accept, get, and reject the ToS for a user" in {
+  "TosService.acceptTosStatus" should "accept the ToS for a user" in {
```

Comment:
```suggestion
  "TosService" should "accept the ToS for a user" in {
```
Something I just read in https://learning.oreilly.com/library/view/unit-testing-principles/9781617296277/ , putting the method name in the test is not recommended since it ties the test name to the implementation.  The name of the method is irrelevant, what we want to assert is the behavior that a user can accept the ToS.

Reply:
TIL!